frontend/src/components/Header/Header.tsx

```diff
@@ -31,10 +31,21 @@ export const Header: React.FC<HeaderProps> = ({
     score,
 }) => {
 
+    // TODO: Fix this permanently and localize in the backend
+    // See also: https://github.com/Amsterdam-Music-Lab/MUSCLE/issues/1151
+    // Get locale from browser (this is meant to be temporary until we have a proper solution for localization in the backend)
+    const locale = navigator.language || 'en-US';
+    const localeIsEnglish = locale.includes('en');
+
+    console.log({ locale })
+
+    // Get current URL minus the query string
+    const currentUrl = window.location.href.split('?')[0];
+
```

Comment:
> Ah, overlooked that the text was set in the frontend. I vote to set this text from the backend, as part of the collection's serializer. This is something we have to do anyway once #1154 is in place.
> 
> The Dutch share message should actually be set to this:
> 
> https://github.com/Amsterdam-Music-Lab/MUSCLE/blob/47975e75a9a1caa1c496b40225d2dc81dec24eb9/backend/experiment/rules/toontjehoger_home.py#L117

For clarification: Are you expecting me to update the collection model and serializer with a `share_message` property or just to update the texts in the frontend for now?

Reply:
Not the model, just the serializer. Let us hold the database changes until we know how to handle localization.

Reply:
The serializer will have to send some kind of "share_message" eventually, and for now it can be `_("I scored {} points.format(score)"`, or ideally a slightly different message if the user didn't score points yet.

Reply:
I don't agree to this. I don't understand why we have to update the serializer, the interfaces in the frontend, the `ExperimentCollection`, the `ExperimentCollectionDashboard`, and the `Header` component for showing a simple hard-coded message for just a hotfix when the definitive solution/fix might look completely different in the future. Additionally, the total score comes from the group serializer, when the share message seems something more related to the collection.